hello

```diff
@@ -0,0 +1,6 @@
+set -o errexit
```

Comment:
This really should have a shebang (`#!/bin/sh`, `#!/bin/bash`, `#!/usr/bin/env bash`, etc), or it won't be executable except by a shell (which will generally fall back to executing something as a script if the `execv` syscall fails).